instance/tests/models/test_openedx_ansible_mixins.py

```diff
@@ -90,7 +90,7 @@ def test_provisioning(
             inventory_str=mock_inventory,
             vars_str=appserver.configuration_settings,
             playbook_path='{}/playbooks'.format(working_dir),
-            playbook_name='edx_sandbox.yml',
+            playbook_name='openedx_native.yml',
```

Comment:
@giovannicimolin, it might be worth testing for the ginkgo, hawthorn releases by passing the release and the expected value via ddt.